tests/helpers.py

```diff
@@ -2101,3 +2101,18 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+def checkLSO():
```

Comment:
There is an existing function csv.get_csvs_start_with_prefix() which returns boolean when the csv_prefix match is found. 
Similar implementation is done in https://github.com/red-hat-storage/ocs-ci/pull/1692/files
snip code:
========
from ocs_ci.ocs.resources import csv
    if csv.get_csvs_start_with_prefix(
        csv_prefix=defaults.LOCAL_STORAGE_OPERATOR_NAME,
        namespace=defaults.LOCAL_STORAGE_OPERATOR_NAMESPACE
    ):
        return True

Can you check if this can be used?